template-react-typescript/src/react-app-env.d.ts

```diff
@@ -0,0 +1,65 @@
+
+/// <reference types="react" />
+/// <reference types="react-dom" />
+
+declare namespace NodeJS {
+  interface ProcessEnv {
+    readonly NODE_ENV: 'development' | 'production' | 'test'
+  }
+}
+
+declare module '*.bmp' {
```

Comment:
Vite doesn't support `bmp`. I don't think anyone should be using `bmp` in web projects nowadays.